application/src/main/java/com/knubisoft/application/ecryption/Encryption.java

```diff
@@ -0,0 +1,10 @@
+package com.knubisoft.application.ecryption;
+
+import java.security.PrivateKey;
+import java.security.cert.X509Certificate;
+
+public interface Encryption {
+
+    byte[] encryptData(byte[] rawData, X509Certificate certificate);
```

Comment:
Can we update encrypt and decrypt method signatures to accept byte[] instead of X509Certificate and PrivateKey?